ambry-server/src/test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -210,8 +210,8 @@ protected static void endToEndTest(Port targetPort, String coordinatorDatacenter
       channel.send(getRequestExpired);
       InputStream streamExpired = channel.receive().getInputStream();
       GetResponse respExpired = GetResponse.readFrom(new DataInputStream(streamExpired), clusterMap);
-      Assert
-          .assertEquals(respExpired.getPartitionResponseInfoList().get(0).getErrorCode(), ServerErrorCode.Blob_Expired);
+      Assert.assertEquals(respExpired.getPartitionResponseInfoList().get(0).getErrorCode(),
```

Comment:
while you are at it, you could switch the params - it is supposed to be `assert(expectedValue, actualValue)` in general and that is the way it will get printed too if it fails.